src/main/java/duke/command/Parser.java

```diff
@@ -0,0 +1,87 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.exception.DukeInvalidCommandException;
+
+/**
+ * A class that deals with making sense of user input.
+ * It creates the appropriate Command object accordingly.
+ */
+public class Parser {
+
+    public static Command parse(String line) throws DukeException {
+        String[] chunked = line.split(" ");
+        String command = chunked[0];
+        if (command.equals("bye")) {
```

Comment:
maybe you can consider using switch statements instead of multiple if else statements with repeated calling of command.equals().